apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/ir/Operation.kt

```diff
@@ -15,6 +15,7 @@ data class Operation(
     val fields: List<Field>,
     val filePath: String,
     val fragmentsReferenced: List<String>,
+    // NOTE: this property is never correctly set!
     val operationId: String
```

Comment:
Could we remove it ?

Reply:
Yep, can do that. One concern is that currently, we have a few places calling
```QueryDocumentMinifier.minify(operation.sourceWithFragments).sha256()```. Given the discussion around different id hash generation, it might make sense to centralize the generation of this ID so we don't accidentally have a mismatch.

Reply:
Removed the property.